lib/Adapter/TolerantParser/Updater/ClassLikeUpdater.php

```diff
@@ -54,41 +55,48 @@ protected function updateProperties(Edits $edits, ClassLikePrototype $classProto
             return;
         }
 
-        $lastProperty = $classMembers->openBrace;
+        $previousMember = $classMembers->openBrace;
```

Comment:
can we add regression tests for this?

Reply:
or is this just presentation? if it's not a functional change then the test is not so important.

Reply:
The name it's just presentation.
But the fact that the blank line before new properties was moved to the template it's functional.
Didn't I update the tests already ?